### ISSUE TITLE
Error: Clerk: Failed to load Clerk

### ISSUE THREAD

**Issue by mackenly** [clerkinc/use-stripe-subscription-demo#2]
Hi, I pulled down the example, configured the env, ran `npm install`, and then ran the dev script but am getting the following error when opening in browser (it's not showing in the terminal):

```
Unhandled Runtime Error
Error: Clerk: Failed to load Clerk

Call Stack
IsomorphicClerk.throwError
node_modules\@clerk\clerk-react\dist\isomorphicClerk.js (353:0)
IsomorphicClerk.loadClerkJS
node_modules\@clerk\clerk-react\dist\isomorphicClerk.js (341:0)
```

Searched, but couldn't find anything and the code where the error is being thrown doesn't point to any obvious problems.

**Comment by mackenly** [comment]
To troubleshoot tried opening in a codespace to rule out any issues with my environment and experienced the same error.

**Comment by perkinsjr** [comment]
This was Fixed with #3 Thanks for reporting.
